ts/src/pro/bitmex.ts

```diff
@@ -1573,6 +1573,9 @@ export default class bitmex extends bitmexRest {
             } else if (table === 'orderBook10') {
                 this.orderbooks[symbol] = this.indexedOrderBook ({}, 10);
             }
+            if (table === undefined || symbol === undefined) {
```

Comment:
we have to do this earlier because we might be trying to access `this.orderbook` with `symbol = None` which breaks the python version.